crates/services/importer/src/config.rs

```diff
@@ -1,4 +1,14 @@
-#[derive(Default, Debug, Clone)]
+#[derive(Debug, Clone)]
 pub struct Config {
+    pub max_block_notify_buffer: usize,
     pub metrics: bool,
 }
+
+impl Default for Config {
+    fn default() -> Self {
+        Self {
+            max_block_notify_buffer: 1 << 10,
```

Comment:
I think the only slow importers would likely be graphql subscriptions if we decide to expose this channel there. In which case if the are more than 1k blocks behind it's probably a stale connection and dropping the messages is fine.

Reply:
Also as long as the consumers only really care about being notified about latest blocks instead of expecting this to be a persistent and guaranteed delivery message system, we could probably even drop this lower.